src/index.ts

```diff
@@ -8,23 +8,41 @@ import {
 class PayMayaSDK {
     private publicKey: string = '';
     private isSandbox: boolean = true;
-    private apiUrl: string = this.isSandbox ? 'https://pg-sandbox.paymaya.com' : 'https://pg.paymaya.com';
-    private formUrl: string = this.isSandbox ? 'https://paymayajs-staging.s3.amazonaws.com/dist/index.html' : 'https://paymayajs.s3.amazonaws.com/dist/index.html';
+    private apiUrl: string = '';
+    private formUrl: string = '';
+    private eventOrigin: string = '';
 
     public init(publicKey: string, isSandbox: boolean) {
         this.publicKey = publicKey;
         this.isSandbox = isSandbox;
+
+        this.apiUrl = this.isSandbox
```

Comment:
Would it be better if we check the sandbox flag in 1 go, and populate the URLs as a set?

i.e.

```
if (isSandbox) {
    // set all sandbox URLs
} else {
   // set all production URLs
}
```